seeq/addons/correlation/utils/_sdl.py

```diff
@@ -71,3 +72,40 @@ def get_workbook_worksheet_workstep_ids(url):
     if 'workstepId' in params:
         workstep_id = params['workstepId'][0]
     return workbook_id, worksheet_id, workstep_id
+
+
+def is_ipv4(string):
+    temp_str = None
+    try:
+        temp_str = string.split("https://")[1]
+    except IndexError:
+        try:
+            temp_str = string.split("http://")[1]
+        except IndexError:
+            print('Unrecognisable URL or IP returned by SPy')
```

Comment:
don't use print statements here. Raise the exception if needed. The print statement will just make it fail ungracefully somewhere else.